dev/bots/test.dart

```diff
@@ -240,7 +240,8 @@ Future<void> _runToolTests() async {
     File(path.join(flutterRoot, 'bin', 'cache', 'flutter_tools.snapshot')).deleteSync();
     File(path.join(flutterRoot, 'bin', 'cache', 'flutter_tools.stamp')).deleteSync();
   }
-  if (noUseBuildRunner) {
+  // reduce overhead of build_runner in the create case.
+  if (noUseBuildRunner || Platform.environment['SUBSHARD'] == 'create') {
```

Comment:
We need to update the logic at https://github.com/flutter/flutter/pull/35839/files#diff-a9d73960c607f511d218217af489e4d5R587 and following for this

Reply:
Not quite sure what you mean

Reply:
There's logic in there to check the subshard and run the right subset of tests using tags.  I don't think it matches the logic in the build runner variant of the method.

Reply:
Oh duh, got it my bad